src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/settings/SettingsPage.test.tsx

```diff
@@ -186,8 +186,195 @@ it("preselects 'Send breach alerts to the affected email address' if that's the
   expect(primaryRadioButton).not.toHaveAttribute("checked");
 });
 
+it("disables breach alert notification options if a user opts out of breach alerts", async () => {
```

Comment:
Looking at this description, shouldn't this test be checking that the primary and affected radio buttons `.not.beInTheDocument()`?